jsonschema-validation.md

```diff
@@ -0,0 +1,1108 @@
+for the Format-Assertion vocabulary, is:
+`https://json-schema.org/draft/next/vocab/format-assertion`. The current IRI for
+the corresponding meta-schema is:
+`https://json-schema.org/draft/next/meta/format-assertion`. Implementing support
+for the Format-Assertion vocabulary is OPTIONAL.
+
+Specifying both the Format-Annotation and the Format-Assertion vocabularies is
+functionally equivalent to specifying only the Format-Assertion vocabulary since
+its requirements are a superset of the Format-Annotation vocabulary.
+
+### Implementation Requirements
+
+The `format` keyword functions as defined by the vocabulary which is referenced.
+
+#### Format-Annotation Vocabulary The value of format MUST be collected as an
+annotation, if the implementation supports annotation collection. This enables
+application-level validation when schema validation is unavailable or
+inadequate.
+
+Implementations MAY still treat `format` as an assertion in addition to an
+annotation and attempt to validate the value's conformance to the specified
+semantics. The implementation MUST provide options to enable and disable such
+evaluation and MUST be disabled by default. Implementations SHOULD document
+their level of support for such validation.[^2]
+
+[^2]: Specifying the Format-Annotation vocabulary and enabling validation in an
+implementation should not be viewed as being equivalent to specifying the
+Format-Assertion vocabulary since implementations are not required to provide
+full validation support when the Format-Assertion vocabulary is not specified.
+
+When the implementation is configured for assertion behavior, it:
+
+- SHOULD provide an implementation-specific best effort validation for each
+  format attribute defined below;
+- MAY choose to implement validation of any or all format attributes as a no-op
+  by always producing a validation result of true;[^3]
+
+[^3]: This matches the current reality of implementations, which provide widely
+varying levels of validation, including no validation at all, for some or all
+format attributes. It is also designed to encourage relying only on the
+annotation behavior and performing semantic validation in the application, which
+is the recommended best practice.
+
+#### Format-Assertion Vocabulary
+
+When the Format-Assertion vocabulary is declared with a value of true,
+implementations MUST provide full validation support for all of the formats
+defined by this specificaion. Implementations that cannot provide full
+validation support MUST refuse to process the schema.
+
+An implementation that supports the Format-Assertion vocabulary:
+- MUST still collect `format` as an annotation if the implementation supports
+  annotation collection;
+- MUST evaluate `format` as an assertion;
+- MUST implement syntactic validation for all format attributes defined in this
+  specification, and for any additional format attributes that it recognizes,
```

Comment:
Other instances of bulleted lists are surrounded by blank lines.  We should at least be consistent.